tools/astarte_export/lib/fetchdata/queries/queries.ex

```diff
@@ -0,0 +1,249 @@
+defmodule Astarte.Export.FetchData.Queries do
+  alias Astarte.Core.InterfaceDescriptor
+  alias Astarte.Core.Mapping
+  require Logger
+
+  def get_connection() do
+    nodes = Application.get_env(:xandra, :cassandra_nodes)
+    {host, port} = Enum.random(nodes)
+    Logger.info("Connecting to #{inspect(host)}:#{inspect(port)} cassandra database.")
+
+    with {:ok, xandra_conn} <- Xandra.start_link(nodes: ["#{host}:#{port}"], atom_keys: true) do
+      Logger.info("Connected to database.")
+      {:ok, xandra_conn}
+    else
+      {:error, reason} ->
+        Logger.error("DB connection setup failed: #{inspect(reason)}", tag: "db_connection_failed")
+    end
+  end
+
+  def retrieve_interface_row(conn, realm, interface, major_version, options) do
```

Comment:
Did you copy paste this code from astarte_data_access?

Reply:
Just naming convention i had used

Reply:
astarte_data_access implemented this function using CQex. i had updated the same with Xandra, as we are migrating from CQex to Xandra